waspc/src/Wasp/Generator/NpmDependencies.hs

```diff
@@ -108,9 +107,10 @@ buildNpmDepsForFullStack spec forServer forWebApp =
 getUserNpmDepsForPackage :: AppSpec -> NpmDepsForUser
 getUserNpmDepsForPackage spec =
   NpmDepsForUser
-    { userDependencies = fromMaybe [] $ AS.App.dependencies $ snd $ ASV.getApp spec,
+    { -- todo(filip): what if package.json has no dependencies field?
```

Comment:
Hm good question. Well that is same as [] right?

Reply:
Yes, but I'm wondering if I should model it with a `Maybe` inside AppSpec (spoiler: yes, probably, but I'll try to break it first).

Reply:
Ah got it! Yeah, you probably should, as that provides a bit more info, but, in practice, I don't think it will ever matter.

Reply:
Covered in https://github.com/wasp-lang/wasp/issues/1846